sys-power/powerstat/powerstat-0.02.11.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Laptop power measuring tool"
+HOMEPAGE="https://launchpad.net/ubuntu/+source/${PN}"
+SRC_URI="http://kernel.ubuntu.com/~cking/tarballs/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Per https://github.com/ColinIanKing/powerstat/blob/master/Makefile#L7 the correct license is `GPL-2+`